src/website/src/pages/ec/components/forms/select/docs/usage.mdx

```diff
@@ -36,27 +38,27 @@ import { Paragraph, Anatomy, Link } from '@ecl/website-components';
 | heading          | yes       | heading describes the options in the Select Menu                                                   |
 | helper text      | no        | helper text information assists a user in using Select Menu                                        |
 | dropdown list    | yes       | dropdown in the collapsed state with placeholder giving instructions for using the Dropdown Select |
```

Comment:
Not sure whether it is to be taken into account, wiki includes a small note here
"For single-select, the design of the expanded container cannot be controlled as it's provided by the operating system of the user"